sys/kern/vfs_aio.c

```diff
@@ -199,7 +199,11 @@ typedef struct oaiocb {
 	struct	osigevent aio_sigevent;	/* Signal to deliver */
 	int	aio_lio_opcode;		/* LIO opcode */
 	int	aio_reqprio;		/* Request priority -- ignored */
-	struct	__aiocb_private	_aiocb_private;
+	struct {
```

Comment:
What happened to our `__aiocb_private`?..

Reply:
I probably deleted it when cheriabi was non-default.  I'll restore it and delete this diff.